File: LabsC++/ifElse/workerBonus.cpp
#include <iostream>
using namespace std;
int main(){
    int age;
    cout << "Those who have worked for at least 5 years now get a $1000 bonus!\n";
    cout << "For how log has this person been working here? (In years) ";
    cin >> age;
    try{
        if(age < 0){
            throw "Employement time can't be negative!";
        }
        else{
            if(age >= 5){
                cout << "This worker can now get the $1000 bonus!";
            }
            else{
                cout << "No bonus for this worker...";
            }
        }
    }
    catch(const char* error){
        cout << "ERROR: " << error;
    }
}

File: LabsC#/workerBonus/workerBonus.cs
// See https://aka.ms/new-console-template for more information
using System;
namespace workerBonus // Note: actual namespace depends on the project name.
{
    internal class workerBonus
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Those who have worked for at least 5 years now get a $1000 bonus!");
            Console.WriteLine("For how long has this person been working for? (In years) ");
            int age = int.Parse(Console.ReadLine());
            if(age >= 5){
                Console.WriteLine("This worker can now get the $1000 bonus!");
            };
            if(age < 5){
                Console.WriteLine("No bonus for this worker...");
            };
        }
    }
}